apps/openmw/mwrender/screenshotmanager.cpp

```diff
@@ -251,14 +251,13 @@ namespace MWRender
 
         osg::ref_ptr<osg::Camera> screenshotCamera(new osg::Camera);
         osg::ref_ptr<osg::ShapeDrawable> quad(new osg::ShapeDrawable(new osg::Box(osg::Vec3(0,0,0), 2.0)));
-        quad->getOrCreateStateSet()->setRenderBinDetails(100, "RenderBin", osg::StateSet::USE_RENDERBIN_DETAILS);
```

Comment:
There's no explanation for why the renderbin for stitching the RTTs is gone. Is there any other mechanism ensuring the RTTs happen before we try and stitch them?

Reply:
This `RenderBin` was not used because we assign another `StateSet` later on.

Reply:
In addition, there is no point to use any `RenderBin` here because its associated `Camera` only contains one `Drawable`.

Reply:
> Is there any other mechanism ensuring the RTTs happen before we try and stitch them?

Yes, they are already copied into `osg::Image` objects before we even call this line.

Reply:
Cool